tests/parametertree/test_parametertypes.py

```diff
@@ -7,8 +7,10 @@
 from pyqtgraph.functions import eq
 from pyqtgraph.parametertree.parameterTypes import ChecklistParameterItem
 from pyqtgraph.Qt import QtCore, QtGui
+from pyqtgraph.parametertree.parameterTypes import SimpleParameter
```

Comment:
SimpleParameter and Element imports are not used.
Probably should just revert the file to its original state.

Reply:
Yes I think so I just revert it